src/autosuggest/internal.tsx

```diff
@@ -115,8 +115,8 @@ const InternalAutosuggest = React.forwardRef((props: InternalAutosuggestProps, r
   );
   const openDropdown = () => !readOnly && setOpen(true);
   const scrollToIndex = useRef<(index: number) => void>(null);
-  const { highlightedOption, highlightedIndex, moveHighlight, resetHighlight, setHighlightedIndex } =
-    useHighlightedOption(filteredItems);
+  const { highlightedOption, highlightedIndex, highlightedType, moveHighlight, resetHighlight, setHighlightedIndex } =
+    useHighlightedOption({ options: filteredItems, isKeyboard });
```

Comment:
Autosuggest shares useHighlightedOption with select/multiselect so that `usingMouse` is changed to `isKeyboard` which is used in select/multiselect to have the same logic.